pyflakes/checker.py

```diff
@@ -322,10 +322,11 @@ class Binding(object):
                 the node that this binding was last used.
     """
 
-    def __init__(self, name, source):
+    def __init__(self, name, source, isUnassigned=False):
         self.name = name
         self.source = source
         self.used = False
+        self.isUnassigned = isUnassigned
```

Comment:
new code should use `snake_case` -- we're slowly migrating over the legacy code which uses the old casing